hypha/apply/funds/workflow.py

```diff
@@ -1104,7 +1268,11 @@ def make_permissions(edit=None, review=None, view=None, withdraw=None):
                 "proposal_almost": _("Accept but additional info required"),
                 "proposal_accepted": _("Accept"),
                 "proposal_rejected": _("Dismiss"),
-                "proposal_withdrawn": _("Withdraw"),
+                "proposal_withdrawn": {
+                    "display": _("Withdraw"),
+                    "permissions": {UserPermissions.APPLICANT},
+                    "method": "withdraw",
+                },
```

Comment:
The previous was more concise in that it set . What is the advantage of this repetition?

Reply:
I could have sworn I tested with applicants but I guess you mean in the context of putting the permissions in a single place rather than multiple places.